readme.md

```diff
@@ -1,18 +1,49 @@
 # Spring PetClinic Sample application [![setup automated](https://img.shields.io/badge/Gitpod-ready_to_code-orange?logo=gitpod)](https://gitpod.io/from-referrer/)
 
-Start coding in a [ready-to-code development environment](https://www.gitpod.io):
+[Petclinic](https://github.com/spring-projects/spring-petclinic) is a [Spring Boot](https://spring.io/guides/gs/spring-boot/) application built using [Maven](https://spring.io/guides/gs/maven/).
 
-<a href="https://gitpod.io/from-referrer/" style="padding: 10px;">
-    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">
-</a>
+## Table of Contents
+
+ - [Understanding the Spring Petclinic application with a few diagrams](#understanding-the-spring-petclinic-application-with-a-few-diagrams)
+ - [Run petclinic on Gitpod](#run-petclinic-on-gitpod)
+ - [Get Started With Your Own Project](#get-started-with-your-own-project)
+ - [Community support](#community-support)
 
 ## Understanding the Spring Petclinic application with a few diagrams
 <a href="https://speakerdeck.com/michaelisvy/spring-petclinic-sample-application">See the presentation here</a>
 
 <img width="1042" alt="petclinic-screenshot" src="https://cloud.githubusercontent.com/assets/838318/19727082/2aee6d6c-9b8e-11e6-81fe-e889a5ddfded.png">
 
+## Run petclinic on Gitpod
+
+Start coding in a [ready-to-code development environment](https://www.gitpod.io)
+
+Setup Gitpod integration with Jetbrains Gateway using the instructions given in this [doc](https://www.gitpod.io/docs/ides-and-editors/intellij).
+
+Click the button below to start a new workspace on Gitpod which will automatically open the workspace in IntelliJ IDEA through JetBrains Gateway.  
```

Comment:
**Thought:** Not sure if we should re-word this somehow...

1. First-time users are presented with an IDE choice (so will have the option for IntelliJ)
2. Second-time users and Gitpod will only open if the user has selected IntelliJ before. 

The latter might be quite common as VS Code Browser is the default, and we only ask on the first workspace. The user then has to ensure that they set their global preference for subsequent workspaces. Until we have hardcode-able URL's to ensure that specific IDEs are opened having this info in the readme might be brittle / get outdated quite quickly. 🤔